### PR TITLE
Added README

### PR DESCRIPTION
**Pull request by tamas-molnar**

### CODE REVIEW

**Comment by Nesze** [comment]
lgtm

File: README.md [hunk 1]
```diff
@@ -1 +1,75 @@
...
+
+**Locks the DB it's run on, archives and compresses the given data folder, uploads it to an AWS S3 bucket, and unlocks the DB.**
+
+## Installation
+
+For getting started with Go, refer to the [nativerw README](https://github.com/Financial-Times/nativerw/blob/master/README.md)
+
+`go install github.com/Financial-Times/coco-mongodb-backup`
+
+:warning: Needs GO 1.5 or higher
```
Comment (ghost) [inline README.md]: "warning" seems a bit strong! :)

File: README.md [hunk 2]
```diff
@@ -1 +1,75 @@
...
+coco/coco-mongodb-backup
+```
+
+## Implementation details
+### Gof3r
+Uses [gof3r](https://github.com/rlmcpherson/s3gof3r) to stream the archived folder to S3.
+### Go Pipe
+Uses a [pipe](https://golang.org/pkg/io/#Pipe) to transfer the output of the archiver straight to S3 - this means that the archived folder itself isn't stored on the disk.
+
+The pipe is synchonous, meaning that the reader cannot read from the pipe until the corresponding writer is writing something, and vice-versa.
```
Comment (ghost) [inline README.md]: this paragraph probably just duplicates documentation on io.Pipe

**Comment by catalin-ilea** [comment]
+1

**Pull request merged**